### ISSUE TITLE
ModuleNotFoundError: No module named 'botorch.models.fidelity'

### ISSUE THREAD

**Issue by ZoranPandovski** [facebook/Ax#237]
Run a simple optimization loop throws ModuleNotFoundError, using ax-platform==0.1.6 version.
```
from ax import optimize
best_parameters, best_values, experiment, model = optimize(
        parameters=[
          {
            "name": "x1",
            "type": "range",
            "bounds": [-10.0, 10.0],
          },
          {
            "name": "x2",
            "type": "range",
            "bounds": [-10.0, 10.0],
          },
        ],
        # Booth function
        evaluation_function=lambda p: (p["x1"] + 2*p["x2"] - 7)**2 + (2*p["x1"] + p["x2"] - 5)**2,
        minimize=True,
    )
```
**Traceback**

> Traceback (most recent call last):
>     from ax.models.torch.botorch_defaults import (
>   File "/home/zoran/MyProjects/mindsdb-examples/pl/lib/python3.7/site-packages/ax/models/torch/botorch_defaults.py", line 12, in <module>
>     from botorch.models.fidelity.gp_regression_fidelity import (
> ModuleNotFoundError: No module named 'botorch.models.fidelity'

It looks like there are changes in the latest botorch 0.2.0 version. Downgrading to 0.1.4 fixes the issue.

**Comment by shichangzhang-coder** [comment]
I also encountered the same problem, hope to be resolved, thanks!

**Comment by shichangzhang-coder** [comment]
Downgrading to 0.1.4 can fixes the issue, thank you ZoranPandovski ^^

**Comment by Balandat** [comment]
Thanks for flagging this. There were some hiccups in the tandem release we had planned for Ax & botorch, for now you can either use the latest Ax release with botorch 0.1.4 or the current master with botorh 0.2.0. 

We're currently working on cutting a new Ax release compatible with 0.2.0 that should be out soon.

**Comment by huazuhao** [comment]
1) What do you mean by current master with botorch 0.2.0? Isn't the pip version already 0.2.0?
2) How exactly do we downgrade to 0.1.4? Should we just checkout a particular historical commit?

**Comment by huazuhao** [comment]
Also, we tried to install the bleeding edge version of Ax but got the error code of 128 during the installation process. Thank you.

**Comment by Balandat** [comment]
1. The botorch pip version is already 0.2.0, correct. You can use that with the Ax master - you can install the Ax master via pip using `pip install git+ssh://git@github.com/facebook/Ax.git` as described in the readme. 

2. You can install the specific botorch version from pip using `pip install -e git://github.com/pytorch/botorch.git@v0.1.4`. Then you can install the Ax release as usual.

**Comment by lena-kashtelyan** [comment]
This should be resolved in latest Ax, 0.1.8. Please reopen if the issue does not go away for you with version update, @ZoranPandovski!

**Comment by microprediction** [comment]
I see to have a similar issue with the "cover page" example .
https://github.com/microprediction/tuneup/blob/master/tests/test_ax.py
